protos/fpc/fpc.proto

```diff
@@ -0,0 +1,36 @@
+// Copyright IBM Corp. All Rights Reserved.
+// Copyright 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0
+
+syntax = "proto3";
+
+import "google/protobuf/any.proto";
+
+package fpc;
+option go_package = "github.com/hyperledger-labs/fabric-private-chaincode/internal/protos";
+
+message CC_Parameters {
+    string chaincode_id   = 1;      // chaincode name
+    string version        = 2;      // chaincode version, this is the expected mrenclave
+    string sequence       = 3;      // chaincode sequence number
+    string channel_id     = 4;      // channel id
+}
+
+message Host_Parameters {
+    bytes peer_identity = 1;        // peer that hosts the enclave
```

Comment:
what is the encoding here? Will this be a serialized msp.SerializedIdentity from Fabric?

Reply:
added a reference to https://github.com/hyperledger/fabric-protos/blob/master/msp/identities.proto#L15